web/src/buddy_list.js

```diff
@@ -216,13 +216,13 @@ export class BuddyList extends BuddyListConf {
 
     insert_new_html(opts) {
         const new_key = opts.new_key;
-        const html = opts.html;
+        const $html = $(opts.html);
```

Comment:
This commit breaks our node tests because `opt.html` is `undefined`. I only added this here because it was [requested](https://github.com/zulip/zulip/pull/20464#discussion_r977042472), although, I would personally just squash this.

Node tests work otherwise.